### ISSUE TITLE
Something Wrong with OSS after the Release of TensorRT 7.1 

### ISSUE THREAD

**Issue by Zenvi** [NVIDIA/TensorRT#665]
Before the release, I was able to convert my RetinaFace FaceDetector to TRT engine file using the method:
[https://github.com/NVIDIA/TensorRT/issues/386#issuecomment-604183816](url)
Yet after the release, I get Segmentation Fault (core dumped) every time when I tried to do the conversion.
`Segmentation fault (core dumped)`
Yet another, creep things happen, when I enable the '--verbose' option in 'trtexec' command, and tried the conversion again, at the bottom of my command line, it gave:
` ----- Parsing of ONNX model /mnt/FaceDetector.onnx is Done ---- 
Segmentation fault (core dumped)`
I think my model has already been parsed but there shall be some internal error with trtexec that's preventing me from seeing it pass.

STEP TO REPRODUCE:
1. Download my RetinaFace detector .onnx file from here: 
[FaceDetector.onnx.gz](https://github.com/NVIDIA/TensorRT/files/4874986/FaceDetector.onnx.gz)
2. `docker run --runtime=nvidia -it -v $PWD:/mnt nvcr.io/nvidia/tensorrt:20.03-py3`
3. `wget https://raw.githubusercontent.com/rmccorm4/tensorrt-utils/master/OSS/build_OSS.sh
source build_OSS.sh`
4. `trtexec --explicitBatch --onnx=/mnt/FaceDetector.onnx`

NOTE:
1. I also tried the 20.02-py3 image, which gave me the same result.
2. I was able to do the parsing and conversion before the release, here's my .engine file:
[RetinaFace_mnet.engine.gz](https://github.com/NVIDIA/TensorRT/files/4874979/RetinaFace_mnet.engine.gz)
3. I also tried the official tutorial's yolov3.onnx file, didn't work also.

I'm just wondering is it only me facing this situation? It's pretty strange that the same steps turned invalid on the same model under the same environment when it was only a night passed...

**Comment by Zenvi** [comment]
@rmccorm4 Summoning the big guy.

**Comment by rmccorm4** [comment]
Hi @Zenvi ,

The `build_OSS.sh` script you referenced will build the master branch of this repo and the onnx parser repo by default. The `nvcr.io/nvidia/tensorrt:20.03-py3` is TensorRT 7.0, but the current master branch of the OSS repo is for TensorRT 7.1. 

You can probably achieve the same behavior you had before by doing something like this (didn't test myself):
```
docker run --runtime=nvidia -it -v $PWD:/mnt nvcr.io/nvidia/tensorrt:20.03-py3
wget https://raw.githubusercontent.com/rmccorm4/tensorrt-utils/master/OSS/build_OSS.sh
BRANCH="release/7.0"
source build_OSS.sh "${BRANCH}"
```

or you could update to the 20.06 image for TensorRT 7.1 and probably be able to use the master branch of OSS again:
```
docker run --runtime=nvidia -it -v $PWD:/mnt nvcr.io/nvidia/tensorrt:20.06-py3
wget https://raw.githubusercontent.com/rmccorm4/tensorrt-utils/master/OSS/build_OSS.sh
BRANCH="release/7.1"
source build_OSS.sh "${BRANCH}"
```

---

Also, the NGC TensorRT containers come with a more official `opt/tensorrt/build_opensource.sh` script you can use as well. I believe the current 20.06 container has a typo in it that mentions "TRT_TAG=20.05" (https://github.com/NVIDIA/TensorRT/issues/682), but you can simply edit that to 20.06 and it should work.

**Comment by Zenvi** [comment]
Hey @rmccorm4 and thank you for your reply!
I tried the source build_OSS.sh "${BRANCH}" command and when building a former version of trtexec, resize ops are just not supported yet.
here's how I solved the issue:
1. docker run --runtime=nvidia -it -v $PWD:/mnt nvcr.io/nvidia/tensorrt:20.02-py3
2. source build_OSS.sh 20.02
3. modify the file in `./TensorRT/parsers/onnx/ModelImporter.cpp`: line: 175, to be `if ((output || output.is_weights()) && !outputName.empty())`
4. exit the container and reenter, rebuild OSS by `source build_OSS.sh 20.02`. And after that my trtexec supports resize and runs smoothly.

Ps: I did try the 20.06 container, yet my gpu is not supportive for driver 450 (or so?), so it failed. Yet anyone whose gpu is supportive please feel free to use the 20.06 container, there should be nothing wrong.

**Comment by rmccorm4** [comment]
Just to update, these issues have been fixed in the 20.07 container:

https://github.com/NVIDIA/TensorRT/issues/699#issuecomment-662114078
